javatests/com/williamfiset/algorithms/search/InterpolationSearchTest.java

```diff
@@ -0,0 +1,38 @@
+package javatests.com.williamfiset.algorithms.search;
```

Comment:
Please make sure you can build and run this file. It looks like there are a few missing semi-colons.